samples/ControlCatalog/ViewModels/ItemsRepeaterPageViewModel.cs

```diff
@@ -55,29 +55,23 @@ private ObservableCollection<Item> CreateItems()
             return new ObservableCollection<Item>(
                 Enumerable.Range(1, 100000).Select(i => new Item(i)
                 {
-                    Text = $"Item {i.ToString()} {suffix}"
+                    Text = $"Item {i} {suffix}"
```

Comment:
This adds extra 100k boxed integers, so it would be nice to keep `i.ToString()` here :)

Reply:
Ah apologies, was just following VS's suggestion here. I'll revert.